crates/ruma-common/src/events/room/message.rs

```diff
@@ -374,7 +376,7 @@ impl InReplyTo {
 }
 
 /// The event this relation belongs to replaces another event.
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Deserialize, Serialize)]
```

Comment:
There is a separate `JsonRepr` type for this, why do we now need to derive `Deserialize` and `Serialize`?